posts/nsf-awards-cantera-project-25m-through-2022.md

```diff
@@ -0,0 +1,18 @@
+---
+- title: NSF awards Cantera Project $2.5M through 2022
+- slug: nsf-awards-cantera-project-25m-through-2022
+- date: '2019-09-09 15:07:07 UTC-04:00'
+- tags: funding
+- category: null
+- link: null
+- description: null
+- type: text
+---
+
+# NSF awards Cantera Project $2.5M through 2022
+
+We are proud to announce that NSF has awarded funds to members of the development team and steering committee, under the _Cyberinfrastructure for Sustained Scientific Innovation (CSSI)_ program. The proposal, titled _Extensible and community-driven thermodynamics, transport, and chemical kinetics modeling with Cantera: expanding to diverse scientific domains_, was awarded $2.5M USD to enable new scientific modeling capabilities, improvements to the Cantera software architecture, and community development activities. These activities will help further Cantera's primary objective, which is a generalizable software package to assist users in a variety of fields with a range of calculations related to thermodynamics, chemical kinetics, and species transport.
```

Comment:
should we make the proposal public? The Jupyter project also just got a big NSF grant, and put their proposal on Zenodo: https://blog.jupyter.org/jupyter-meets-the-earth-1b0eb33c83f

Reply:
We should discuss this in the group. We can update the blog post (or make a new one) if we decide to release it.